packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -431,7 +433,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
           }
           this.suggestionStore.removeSuggestion(this.suggestionStore.currentIndex);
           this.forceUpdate();
-        } else {
+        } else if(!this.props.disabled) {
```

Comment:
shouldn't we just skip this entire if statement if it is disabled? Is there a situation where a disabled picker would need to remove a suggestion?

Reply:
I agree, this should be first.

Reply:
That makes sense. My bad I missed it. 
Floated a new iteration.